components/prize_pool/commons/prize_pool.lua

```diff
@@ -537,6 +544,59 @@ function PrizePool:setLpdbInjector(lpdbInjector)
 	return self
 end
 
+function PrizePool:_storeLpdb()
+	local lpdbTournamentData = {
+		tournament = Variables.varDefault('tournament_name'),
+		parent = Variables.varDefault('tournament_parent'),
+		series = Variables.varDefault('tournament_series'),
+		shortname = Variables.varDefault('tournament_tickername'),
+		startdate = Variables.varDefaultMulti('tournament_startdate', 'tournament_sdate', 'sdate', ''),
+		mode = Variables.varDefault('tournament_mode'),
+		type = Variables.varDefault('tournament_type'),
+		liquipediatier = Variables.varDefault('tournament_liquipediatier'),
+		liquipediatiertype = Variables.varDefault('tournament_liquipediatiertype'),
+		-- publishertier = ,
+		icon = Variables.varDefault('tournament_icon'),
+		icondark = Variables.varDefault('tournament_icondark'),
+		game = Variables.varDefault('tournament_game'),
+	}
+
+	local setWeight = function(lpdbEntry)
+		lpdbEntry.weight = Template.safeExpand(
+			mw.getCurrentFrame(), 'Weight', {
+				math.max(lpdbEntry.prizemoney, 1),
```

Comment:
what about prize values between 0 and 1?

Reply:
This is how the old Prize Pool worked. Using the other factors instead of PrizeMoney, as the weight when PrizeMoney < 1.

Reply:
sc uses the money value as is without any minimum 
sc2 has 0.0001 as minimum

Reply:
Ok, I should have said for many wikis. However, it's rare enough that a prizepool is between 0 and 1 USD, and to be fair, all things considered, it's not particular relevant for the weight calculations. A tournament where a participant got a $0.5 cash is frankly just as much an achievement as if they had gotten $0.

Reply:
alright